src/frontend/js/components/ReferralActivityIndicator/index.tsx

```diff
@@ -92,6 +92,11 @@ const messages = defineMessages({
       'Activity indicator message for a validation request on a referral answer draft.',
     id: 'components.ReferralActivityIndicator.validationRequested',
   },
+  [ReferralActivityVerb.URGENCYLEVEL_CHANGED]: {
+    defaultMessage: '{ actorName } changed the urgency level',
```

Comment:
Ici on pourrait peut-être plutôt parler de délai de réponse attendu? `changed the expected response date`